raitools/raitools/raianalyzer/raianalyzer.py

```diff
@@ -0,0 +1,69 @@
+# Copyright (c) Microsoft Corporation
+# Licensed under the MIT License.
+
+"""Defines the RAIAnalyzer class."""
+
+from raitools._managers.causal_manager import CausalManager
+from raitools._managers.counterfactual_manager import CounterfactualManager
+from raitools._managers.error_analysis_manager import ErrorAnalysisManager
+from raitools._managers.explainer_manager import ExplainerManager
+from raitools._managers.fairness_manager import FairnessManager
+
+
+class RAIAnalyzer(object):
+    def __init__(self, model, train, test, target_column,
+                 task_type, classes=None):
```

Comment:
Is `classes` used by anything other than Explainer? If not, shouldn't it be on the Explainer's `add()` method?

Reply:
it's used by error analysis too, I'm not sure if it is used by anything else, but it's an optional parameter (really it's only used for the visualizations, it's not used by the explainer and is just passed onto the explanation object)

Reply:
Have we figured out how to make a useful 'static' version of `ErrorAnalysis`? I wasn't sure if it was included in this yet. If it's only used for visualisation, when does it actually need to be supplied? How do we get from this object to starting up a Model Assessment Dashboard in a notebook?

Reply:
it's an optional argument that is not required.  I have a feeling it will be used by other libraries too though.  I'm surprised fairlearn doesn't use it.  Do you just say "for class 1" instead of "for class >=50k" in the adult census income dataset for fairlearn dashboard, for example?  It's much easier to understand when you use class names.

Reply:
The class names are just the unique values of the given sensitive feature column.

Reply:
Oh I see, I can remove this parameter then and assume that the classes are the unique values in the target_column of the training dataset, we can just really limit the scope for now

Reply:
I removed the parameter from RAIAnalyzer, and I just compute the classes based on the unique values in the target column on the training dataset now as you suggested.  In some cases models can't handle string classes though and user might want to keep them separate from training dataset, so I feel like long-term we will probably have to add it back as an optional parameter.